libraries.json

```diff
@@ -13,53 +19,212 @@
+		"language": "JavaScript"
+	},
+	"Redux": { 
+		"imports": ["redux"],
+		"technologies": ["Web Development"],
+		"description": "A predictable state container for JavaScript apps.",
+		"image": "????",
+		"language": "JavaScript"
+	},
+	"Rollup": {
+		"imports": ["rollup"],
+		"technologies": ["Build tools"],
+		"description": "Parcel is a web application bundler, differentiated by its developer experience. It offers blazing fast performance utilizing multicore processing, and requires zero configuration.",
+		"language": "JavaScript"
+	},
+	"RxJS": {
+		"imports": ["rxjs"],
+		"technologies": ["Web Development"],
+		"description": "Reactive Extensions Library for JavaScript",
+		"image": "????",
+		"language": "JavaScript"
+	},
+		"Svelte": {
 		"imports": ["svelte"],
 		"technologies": ["Web Development", "SPA"],
 		"description": "Svelte is a radical new approach to building user interfaces. Whereas traditional frameworks like React and Vue do the bulk of their work in the browser, Svelte shifts that work into a compile step that happens when you build your app.",
 		"image": "https://raw.githubusercontent.com/sveltejs/branding/master/svelte-logo.png",
 		"language": "JavaScript"
 	},
+	"ThreeJS": {
+		"imports": ["three"],
+		"technologies": ["WebGL"],
+		"description": "DescriptionThree.js is a cross-browser JavaScript library and Application Programming Interface used to create and display animated 3D computer graphics in a web browser.",
+		"language": "JavaScript"
+	},
+	"Universal": {
+		"imports": ["@nguniversal/"],
+		"technologies": ["Web Development", "SPA", "Server side rendering"],
+		"description": "A technology that renders Angular applications on the server.",
+		"language": "JavaScript"
+	},
 	"VueJS": {
 		"imports": ["vue"],
-		"technologies": ["Web Development", "SPA"],
+		"technologies": ["Web Development", "SPA", "JavaScript Frameworks"],
 		"description": "The Progressive JavaScript Framework",
 		"image": "https://raw.githubusercontent.com/vuejs/art/master/logo.png",
 		"language": "JavaScript"
 	},
-	"Go Micro": {
-		"imports": ["github.com/micro/go-micro"],
-		"technologies": ["microservice"],
-		"description": "Go Micro provides the core requirements for distributed systems development including RPC and Event driven communication.",
-		"image": "",
-		"language": "Go"
+	"VueX": {
+		"imports": ["vuex"],
+		"technologies": ["Web Development", "SPA", "JavaScript Frameworks"],
+		"description": "Centralized State Management for Vue.js.",
+		"language": "JavaScript"
+	},
+	"Webpack": {
+		"imports": ["webpack"],
+		"technologies": ["Build tools"],
+		"description": "DescriptionWebpack is an open-source JavaScript module bundler. It is a module bundler primarily for JavaScript, but it can transform front-end assets like HTML, CSS, and images if the corresponding loaders are included.",
```

Comment:
```suggestion
		"description": "Webpack is an open-source JavaScript module bundler. It is a module bundler primarily for JavaScript, but it can transform front-end assets like HTML, CSS, and images if the corresponding loaders are included.",
```